src/multi_stage/write_multi_stage_costs.jl

```diff
@@ -46,6 +52,7 @@ function write_multi_stage_costs(outpath::String, settings_d::Dict, inputs_dict:
 
     # Remove "cTotal" from results (as this includes Cost-to-Go)
     df_costs = df_costs[df_costs[!, :Costs] .!= "cTotal", :]
+    println("**WARNING** : The cost calculation of the multi-stage GenX is approximate currently, and we will be refining it more in one of the future releases.")
```

Comment:
You could also use the `@warn` macro to issue a warning together with or in place of the print statement 
```suggestion
    @warn("The cost calculation of the multi-stage GenX is approximate currently, and we will be refining it more in one of the future releases.")
```